docs/GettingStartedDocs/Contributors/WindowsSGX1GettingStarted.md

```diff
@@ -7,16 +7,18 @@ Intel® X86-64bit architecture with SGX1.
 Note: To check if your system has support for SGX1, please look [here](../SGXSupportLevel.md).
 
 A version of Windows OS with native support for SGX features:
-- For server: Windows Server 2016
+- For server: Windows Server 2016 or 2019
 - For client: Windows 10 64-bit version 1709 or newer
+- To check your Windows version, run `winver` from the command line
 
 ## Install Git and Clone the Open Enclave SDK repo
 
 - Download and install Git for Windows from [here](https://git-scm.com/download/win).
-- Clone the Open Enclave SDK to folder of your choice. In these instructions
-  we're assuming `openenclave`.
+- Clone the Open Enclave SDK to a folder of your choice. In these instructions
+  we're assuming `C:/Users/test`.
 
 ```powershell
+cd C:/Users/test/
```

Comment:
Just for my understanding, why do we specify a longer path here now? Were folks encountering issues with just using the project rooted in C:\?

Reply:
In WindowsSGX1FLCGettingStarted.md, the path was change during 0.8 to c:/users/test. I believe the reason was that the  clone path was  c:\openenclave and then the package also got installed to c:\openenclave which created a lot of confusion. Changing it here to c:\users\test is good since it keeps both docs consistent.